psalm-baseline.xml

```diff
@@ -0,0 +1,348 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<files psalm-version="4.17.0@6f4707aa41c9174353a6434bba3fc8840f981d9c">
+  <file src="config/pipeline.php">
```

Comment:
I wonder if it's possible to fix any of these :thinking:

Reply:
I actually fixed quite a few (spent a few hours on it this morning). The vast majority (as in all but maybe 5-10) were MixedAssignment-style warnings. This is to be expected, as we're testing how the application consumes our _configuration_ and/or the Composer file itself. The amount of effort required really would be better spent on creating a different configuration system that had a schema or type system built-in; anything else is just playing whack-a-mole at this point.